orttraining/orttraining/test/python/orttraining_test_checkpoint_storage.py

```diff
@@ -215,7 +214,7 @@ def test_checkpoint_storage_for_custom_user_dict_succeeds(checkpoint_storage_tes
         'custom_class': custom_class
     }
 
-    pickled_bytes = binascii.b2a_hex(pickle.dumps(user_dict))
+    pickled_bytes = pickle.dumps(user_dict).hex()
```

Comment:
I noticed you changed the implementation. binascii was described as supper efficent and low level. did you change to pickle for compatibility or something?

Reply:
even binascii  requires pickle. Pickle serializes the data and binascii b2a_hex would convert the byte string to a hex ascii string. But the b2a_hex function is available natively using byte string ```.hex()``` function. So, no need for the dependency anymore.